src/main/resources/kafka.properties

```diff
@@ -1,19 +1,16 @@
 bootstrap.servers=localhost:9092
-
```

Comment:
Would be useful to keep the spaces in the properties if possible, to help readability.